### ISSUE TITLE
training/validaiton loss/accuracy

### ISSUE THREAD

**Issue by Saeid-Sli** [divamgupta/image-segmentation-keras#295]
How can I have the loss and accuracy of the training and validation phase across all epochs? I want to plot them versus epochs.
Thanks

**Comment by andrey-bat** [comment]
Try callbacks or parse text output of train.
History dataframe not supported in this module.

**Comment by ujlaki15** [comment]
I am trying to figure this out. Leaving a comment here, to increase my chances for success.

**Comment by varungupta31** [comment]
Found any solution?

**Comment by varungupta31** [comment]
Found it. Upon calling, `model.fit` you can store what the call return in a variable, and call `.history` on that object.
Alternatively, you may dump it's contents in a pickle file, and then load it back to get a dictionary containing validation/train - loss + accuracy.
